src/BulletChartSettingsModel.ts

```diff
@@ -23,6 +26,121 @@ export const BulletChartObjectNames = {
     Bullet: { name: BarRectType.Bullet, displayName: "Bullet" },
 } as const;
 
+export const labelsReference: {
+    cardUid: string;
+    groupUid: string;
+    fontFamily: FormattingId;
+    bold: FormattingId;
+    italic: FormattingId;
+    underline: FormattingId;
+    fontSize: FormattingId;
+    labelColor: FormattingId;
+    show: FormattingId;
+} = {
+    cardUid: "Visual-labels-card",
+    groupUid: "labels-group",
+    fontFamily: {
+        objectName: BulletChartObjectNames.Labels.name,
+        propertyName: "fontFamily"
+    },
+    bold: {
+        objectName: BulletChartObjectNames.Labels.name,
+        propertyName: "fontBold"
+    },
+    italic: {
+        objectName: BulletChartObjectNames.Labels.name,
+        propertyName: "fontItalic"
+    },
+    underline: {
+        objectName: BulletChartObjectNames.Labels.name,
+        propertyName: "fontUnderline"
+    },
+    fontSize: {
+        objectName: BulletChartObjectNames.Labels.name,
+        propertyName: "fontSize"
+    },
+    labelColor: {
+        objectName: BulletChartObjectNames.Labels.name,
+        propertyName: nameof<LabelsCard>("labelColor")
+    },
+    show: {
+        objectName: BulletChartObjectNames.Labels.name,
+        propertyName: nameof<LabelsCard>("show")
```

Comment:
why not just "show"?

Reply:
Typescript checks that `LabelsCard` actually has the property named "show" and returns error if there is no such property.